apertium-swe-eng.swe-eng.dix

```diff
@@ -11077,43 +11077,77 @@
 
        
       <e><p><l>akut<s n="adv"/></l><r>acutely<s n="adv"/></r></p></e>
+      <e><p><l>akut<s n="adv"/></l><r>urgently<s n="adv"/></r></p></e>
       <e><p><l>alltmera<s n="adv"/></l><r>increasingly<s n="adv"/></r></p></e>
       <e><p><l>allvarligen<s n="adv"/></l><r>earnestly<s n="adv"/></r></p></e>
       <e><p><l>andligen<s n="adv"/></l><r>spiritually<s n="adv"/></r></p></e>
       <e><p><l>annorlunda<s n="adv"/></l><r>differently<s n="adv"/></r></p></e>
+      <e><p><l>annorlunda<s n="adv"/></l><r>distinctive<s n="adv"/></r></p></e>
```

Comment:
Isn't this an adverbial reading ? so "distinctively" ?

Reply:
Fixed.  😀